lua/gitlab/actions/assignees_and_reviewers.lua

```diff
@@ -38,7 +38,7 @@ M.add_popup = function(type)
     table.insert(current_ids, choice.id)
     local body = { ids = current_ids }
     job.run_job("/mr/" .. type, "PUT", body, function(data)
-      vim.notify(data.message, vim.log.levels.INFO)
+      u.notify(data.message, vim.log.levels.INFO)
```

Comment:
Adding this prefixes all messages with `gitlab.nvim:` which is great for usability IMO